modules/tinymce/CHANGELOG.md

```diff
@@ -26,6 +26,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `InsertParagraph` or `mceInsertNewLine` commands did not delete the current selection like the native command used to #TINY-8606
 - When triple clicking the selection was incorrectly collapsed in the Chrome browser when clicking around nested noneditable content #TINY-8215
 - When pressing the right arrow key, the caret incorrectly moved before any selected inline boundary element #TINY-8601
+- Indenting or outdenting list items inside a block element inside a list item would not work #TINY-8068
```

Comment:
```suggestion
- Indenting or outdenting list items inside a block element that was inside another list item would not work #TINY-7209
```
I believe this was for TINY-7209 instead?

Reply:
I this how we normally do things when there are multiple tickets that gets resolve in one PR. I linked them together in JIRA thought that was enough?

Reply:
Yes this is what we normally do if multiple issues are fixed by the one change (we did the same recently in 6.0.2). Remember the Jira reference is to find out why we made the change if we need to look back in future, so having it point the actual Jira is more meaningful.